README.md

```diff
@@ -552,8 +552,8 @@ Make sure before proceeding you have [pushed all your code to github](https://me
 
 We will now deploy your dApp, so that everyone can see your website and you can share it with all of your LearnWeb3 DAO friends.
 
-- Go to [Vercel](https://vercel.com/) and sign in with your GitHub
-- Then click on `New Project` button and then select your Whitelist dApp repo
+- Go to [Vercel](https://vercel.com/) and sign in with your GitHub account
+- Then click on `New Project` button and select your Whitelist dApp repo
```

Comment:
can you add your discord id as well? We can give you the early contributor role on discord?